cime_config/cam_autogen.py

```diff
@@ -435,18 +435,23 @@ def generate_physics_suites(build_cache, preproc_defs, host_name,
     if not os.path.exists(physics_blddir):
         os.makedirs(physics_blddir)
     # End if
-    # Collect all source directories
-    atm_phys_src_dir = os.path.join(atm_root, "src", "physics", "ncar_ccpp")
-    source_search = [source_mods_dir, atm_phys_src_dir]
-    # Find all metadata files, organize by scheme name
+    # Set top-level CCPP physics directory
+    atm_phys_top_dir = os.path.join(atm_root, "src", "physics", "ncar_ccpp")
+    # Collect all possible Suite Definition File (SDF) locations
+    atm_suites_path      = os.path.join(atm_phys_top_dir, "suites")
+    atm_test_suites_path = os.path.join(atm_phys_top_dir, "test", "test_suites")
+    suite_search         = [source_mods_dir, atm_suites_path, atm_test_suites_path]
+    # Find all scheme metadata files, organized by scheme name
+    atm_schemes_path = os.path.join(atm_phys_top_dir, "schemes")
+    source_search    = [source_mods_dir, atm_schemes_path]
```

Comment:
do we want to include (at the end) the path to the `test` directory in case there are test schemes in there?

Reply:
For now I am leaning towards no, because in an ideal world I would like all physics source code to be either under a case's `SourceMods` directory or the atmospheric_physics' `schemes` directory and associated subdirectories.  Of course if you disagree, or know of a use case where this "ideal" wouldn't work, then please let me know!